.cci.jenkinsfile

```diff
@@ -52,14 +52,15 @@ pod(image: 'registry.fedoraproject.org/fedora:32', runAsUser: 0, kvm: true, memo
 
       stage("Image tests") {
         try {
-          shwrap("cd /srv && rm tmp/kola -rf")
-          shwrap("cd /srv && env TMPDIR=\$(pwd)/tmp/ kola testiso -S")
-          shwrap("cd /srv && env TMPDIR=\$(pwd)/tmp/ kola testiso -S --scenarios iso-offline-install")
-          // and again this time with the 4k image
-          shwrap("cd /srv && env TMPDIR=\$(pwd)/tmp/ kola testiso -S --qemu-native-4k --no-pxe")
+          parallel metal: {
+              shwrap("cd /srv && env TMPDIR=\$(pwd)/tmp/ kola testiso -S --output-dir tmp/kola-testiso-metal")
+          }, metal4k: {
+              shwrap("cd /srv && env TMPDIR=\$(pwd)/tmp/ kola testiso -S --output-dir tmp/kola-testiso-metal4k --no-pxe")
```

Comment:
For posterity, note we lost `--qemu-native-4k` here. It was re-added in https://github.com/coreos/coreos-assembler/pull/1613.